docs/understand-elementary/cli-commands.mdx

```diff
@@ -61,6 +61,13 @@ Here is a list of all the available options:
 - **Configuration directory path:**
   The path of the directory in which your `config.yml` file is located. Default is `HOME_DIR/.edr`.
 
+```shell
+-tp, --target-path <PATH>
+```
+
+- **Configuration target directory path**
+  The path where all output files will be saved, default will create "edr_files" directory in your path
```

Comment:
@noaKurman 
Can they change the default? 
If not I would change to:
The path where `edr` should create an `edr_files` directory where all output files will be saved.

Reply:
Not completely sure what you meant:
You can either run `--target-path ./noa-target` and it would create `noa-target` in the current directory and save all files there, or you can ignore the flag and it would create `edr_files` in your current directory and save all files there, so should I change it to the phrase you suggested? 
Also should I mention it's an absolute path in the doc?
@Maayan-s

Reply:
Ok I get it now! 

So I would write like this:
The absolute path where all output files will be saved. If not configured, the default is `./edr_files`.